legal-api/src/legal_api/models/user.py

```diff
@@ -40,6 +40,11 @@ class User(db.Model):
     iss = db.Column(db.String(1024))
     creation_date = db.Column(db.DateTime(timezone=True), default=datetime.utcnow)
 
+    @classmethod
+    def find_by_id(cls, submitter_id: int = None):
+        """Return a User if they exist and match the provided JWT."""
+        return cls.query.filter_by(id=submitter_id).one_or_none()
+
```

Comment:
Your comment said it checks against JWT - is that true? I don't see that.

Reply:
Oh oops that must have been a copy paste error. I'll update the comment!